src/main/java/ix/ibm/sonar/java/checks/sling/TryWithResourcesResourceResolverRule.java

```diff
@@ -94,7 +85,7 @@ private boolean verifyResourceResolverAccess(final VariableTree tree) {
     }
 
     private boolean isResourceResolverType(final VariableTree tree) {
-        return SLING_RESOURCE_RESOLVER.equals(tree.type().symbolType().fullyQualifiedName());
+        return PackageConstants.SLING_RESOURCE_RESOLVER.equals(tree.type().symbolType().fullyQualifiedName());
```

Comment:
I personally prefer avoiding static imports for readability unless the constant is very often used or the class name is very long. Since we have both approaches in the project we can discuss this and maybe apply changes to the whole project instead of just this one place.